bokehjs/src/coffee/models/glyphs/annular_wedge.coffee

```diff
@@ -107,6 +107,14 @@ class AnnularWedgeView extends Glyph.View
   draw_legend: (ctx, x0, x1, y0, y1) ->
     @_generic_area_legend(ctx, x0, x1, y0, y1)
 
+  _scxy: (i) ->
+    r = (@sinner_radius[i] + @souter_radius[i])/2
+    a = (@_start_angle[i]  + @_end_angle[i])   /2
+    return {x: @sx[i] + r*Math.cos(a), y: @sy[i] + r*Math.sin(a)}
```

Comment:
Seems so

> On May 26, 2016, at 06:11, Mateusz Paprocki <notifications@github.com> wrote:
> 
> In bokehjs/src/coffee/models/glyphs/annular_wedge.coffee:
> 
> > @@ -107,6 +107,14 @@ class AnnularWedgeView extends Glyph.View
> >    draw_legend: (ctx, x0, x1, y0, y1) ->
> >      @_generic_area_legend(ctx, x0, x1, y0, y1)
> >  
> > +  _scxy: (i) ->
> > +    r = (@sinner_radius[i] + @souter_radius[i])/2
> > +    a = (@_start_angle[i]  + @_end_angle[i])   /2
> > +    return {x: @sx[i] + r*Math.cos(a), y: @sy[i] + r*Math.sin(a)}
> @bryevdv, I presume I forgot to consider direction here?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>